website/client/js/controllers/memberModalCtrl.js

```diff
@@ -19,8 +19,10 @@ habitrpg
           $scope.profile = member;
         }
       });
-      
-      $scope.formatAnimal = Costume.formatAnimal;
+
+      for (var method in Costume) {
```

Comment:
@khipkin As I'm going to be using all of the methods exported by the modules, should I just use `var $scope.costume = Costume;`?